client/components/forms/form-phone-input/index.jsx

```diff
@@ -57,7 +57,7 @@ export class FormPhoneInput extends React.Component {
 			<div className={ classnames( this.props.className, 'form-phone-input' ) }>
 				<FormFieldset className="form-fieldset__country">
 					<FormLabel htmlFor="country_code">
-						{ this.props.translate( 'Country Code', {
+						{ this.props.translate( 'Country code', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.